.changeset/green-keys-marry.md

```diff
@@ -0,0 +1,17 @@
+---
+"@localyze-pluto/components": major
+---
+
+UtilityCard:
+
+Breaking Change:
+
+- Removed the `interactiveElementType` prop in order to add two new states properties `hoverable` and `clickable`
+- Renamed `categoryTag` to `subTitle` to make it more generic
+- `onClick` prop is only required when using `clickable` as true
+
+Minor:
```

Comment:
is it fine that this is a major update but we include minor updates?

Reply:
From what we have [here](https://semver.org/#spec-item-8), I would say it's fine. Do you see any specific issue with it?